sdks/python/pom.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.apache.beam</groupId>
+    <artifactId>beam-sdks-parent</artifactId>
+    <version>0.2.0-incubating-SNAPSHOT</version>
```

Comment:
In an early community [discussion](https://lists.apache.org/thread.html/3b201b523701df077bee1a916a8af8dbaf3b11c28aa83015f71dad93@1455032762@%3Cdev.beam.apache.org%3E), we agreed on aligning the versions:

> Finally, we propose the standard approach where the entire source code lives in each branch and is released concurrently. We’d like to avoid the case where individual modules are released on different cadences and are being maintained in different branches of the main repository. This is beneficial because we don’t need to worry as much about versioning some of the surfaces between components. However, for components that have a stable surface or across languages, we can relax this, as appropriate. Additionally, this can be relaxed for hotfixes and different SDK languages.

If we want to start a discussion about relaxing things, we should do so on the dev@ list for visibility.